src/deployment/flavors.md

```diff
@@ -3,41 +3,159 @@ title: Creating flavors for Flutter
+3. Add the display name to info.plist.        
+Create a new key in info.plist called **Bundle Display Name** 
+with the value `$(PRODUCT_NAME)`. 
+![Step 2 using flavors image.](/assets/images/docs/flavors/step2-using-flavors.png){:width="100%"}    
+
+Now you have set up your development flavor by making a `development` scheme 
+in Xcode and setting the build configurations for that scheme. 
+
+For more information, skip to the [“Launching your app flavors”][] 
+section at the end of this document.
+
+## Using flavors in Android
+
+Setting up flavors in Android can be done in your projects 
+**build.gradle** file.
+
+1. In your IDE, inside your Flutter project, 
+navigate to **android** > **app** > **build.gradle**.   
+
+![Step 1 using flavors in Android image.](/assets/images/docs/flavors/android-step1.png){:width="100%"}    
+
+Add a flavor object with the specified environments along with values for 
+**dimension**, **resValue**, and **applicationId**. 
+The name of the application for each build is located in **resValue** 
+(as shown in the screenshot below from VSCode).
+## Setting up launch configurations
+
+In VSCode, set up the launch configurations as follows:  
+Adding a **launch.json** file allows you to run the terminal command 
+`flutter run --flavor [environment name]`. 
+1. In the root directory of your project, add a folder called **.vscode**.    
+2. Inside the **.vscode** folder, create a file named **launch.json**.    
+3. In the **launch.json** file, add a configuration object for each flavor. 
+Each configuration has a **name**, **request**, **type**, **program**, 
+and **args** key.
+
+You can now run the terminal command 
+`flutter run --flavor [environment name]` or you can set up a run 
+configuration in your IDE.
+
+![Image for VSCode launch configuration step.](/assets/images/docs/flavors/vscode-config.png){:width="100%"}  
+
+## Launching your app flavors
+
+1. Once the flavors are set up, modify the Dart code in 
+**lib** > **main.dart** to consume the flavors. 
+2. Test the setup using `flutter run --flavor [environment name]` 
+at the command line, or by setting up a run configuration in your IDE. 
+
+For examples of build flavors for [iOS(Xcode)][] and [Android][], 
+check out the integration test samples in the [Flutter repo][]. 
+
+With this document you were able to: 
+* Set up flavors in iOS by creating a scheme and 
+modifying the build configurations. 
+* Set up flavors in Android by modifying the **build.gradle** file.
+* Set up launch configurations for VSCode.
+* See examples of flavor use in production apps. 
+
+## More information
+
+* [Build flavors in Flutter (Android and iOS) with different Firebase projects per flavor Flutter Ready to Go][]
+* [Flavoring Flutter Applications (Android & iOS)][]
+* [Flutter Flavors Setup with multiple Firebase Environments using FlutterFire and Very Good CLI][]
+
+### Packages
+The following packages are listed alphabetically: 
```

Comment:
It's not clear what the developer is supposed to do with these packages, they would be unnecessary after following the above instructions.
Plus there's only two so we wouldn't need to point out that it's alphabetical 🙂